edx_sysadmin/api/test_views.py

```diff
@@ -115,3 +116,109 @@ def test_GitReloadAPIView(
         )
 
         self.assertEqual(response.status_code, status)
+
+    @override_settings(SYSADMIN_GITHUB_WEBHOOK_KEY=SYSADMIN_GITHUB_WEBHOOK_KEY)
+    @override_settings(SYSADMIN_DEFAULT_BRANCH="master")
+    @patch("edx_sysadmin.api.views.get_local_course_repo", return_value=None)
+    @patch("edx_sysadmin.api.views.add_repo")
+    @ddt.data(
+        (
+            "d3a2424a1ad48d8441712400fd75392d56707a7b3e1dc4869239d87ee381cfa9",
```

Comment:
does that hash represent a commit or just a dummy or so?

Reply:
It represents a sha256 digestible value generated based on the payload. [ref](https://github.com/mitodl/edx-sysadmin/blob/main/edx_sysadmin/api/permissions.py#L65). Also more details on https://github.com/mitodl/edx-sysadmin/pull/63#discussion_r782179751

Reply:
This has to be updated in order for tests to pass since the payload was changed.